### DISCUSSION TITLE
멀티플레이 제스쳐 동기화

### DISCUSSION THREAD

**Discussion by choramha** [naverz/zepeto-studio-kor#931] [5. Q&A]
멀티서버에서 상호작용 버튼을 눌렀을때, 캐릭터가 제스쳐를 취하게 만들고싶습니다.
현재는 호스트 화면에선 모션이 제대로 보이는데, 클라이언트 화면에선 그저 가만히 있는거로 보입니다. 
SetGesture 는 어떤식으로 서버에 전송해야하나요? 전혀 감이 안잡히네요 ㅜㅜ

**Comment by YSHFM** [comment]
호스트에서 제스처 실행하면 키값이던 어떤값이던 애니메이션의 정보와 sessionid를 서버로 전송하고
서버에서 다른 플레이어에게 값을 전달한 뒤 해당 클라이언트에서 전송한 플레이어의 캐릭터에게 SetGesture하면 될 것 같네요.

**Reply by choramha** [reply]
잘 이해가 안가는데 혹시 예제 코드같은거 알려주실 수 있으신가요 ㅜㅜ?

**Reply by YSHFM** [reply]
https://www.mettit.com/pt/view/PRE20220523142110737
https://www.mettit.com/pt/view/PRE20220527110346437

씬동기화와 점프 애니메이션 코드입니다만 이것을 활용하면 서버 송수신에 좀 도움이 될 거 같구요.

https://www.mettit.com/pt/view/PRE20220527165937414

애니메이션인 이부분을 활용하시면 조금 도움이 되실 것 같습니다.
